kafka-ui-react-app/src/components/common/ConfirmationModal/ConfirmationModal.styled.tsx

```diff
@@ -1,58 +1,60 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { Colors } from 'theme/theme';
 
-export const ConfirmationModalWrapper = styled.div.attrs({ role: 'dialog' })`
-  display: flex;
-  align-items: center;
-  flex-direction: column;
-  justify-content: center;
-  overflow: hidden;
-  position: fixed;
-  z-index: 40;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  top: 0;
-  & div:first-child {
-    background-color: rgba(10, 10, 10, 0.1);
+export const ConfirmationModalWrapper = styled.div.attrs({ role: 'dialog' })(
+  ({ theme }) => css`
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    overflow: hidden;
+    position: fixed;
+    z-index: 40;
     bottom: 0;
     left: 0;
-    position: absolute;
     right: 0;
     top: 0;
-  }
+    & div:first-child {
+      background-color: ${theme.confirmationModal.overlay};
+      bottom: 0;
+      left: 0;
+      position: absolute;
+      right: 0;
+      top: 0;
+    }
 
-  & div:last-child {
-    position: absolute;
-    display: flex;
-    flex-direction: column;
-    width: 560px;
-    border-radius: 8px;
+    & div:last-child {
+      position: absolute;
+      display: flex;
+      flex-direction: column;
+      width: 560px;
+      border-radius: 8px;
 
-    background-color: ${Colors.neutral[0]};
-    filter: drop-shadow(0px 4px 16px rgba(0, 0, 0, 0.1));
+      background-color: ${Colors.neutral[0]};
+      filter: drop-shadow(0px 4px 16px ${theme.confirmationModal.shadow});
 
-    & > * {
-      padding: 16px;
-      width: 100%;
-    }
+      & > * {
+        padding: 16px;
+        width: 100%;
+      }
 
-    & > header {
-      height: 64px;
-      font-size: 20px;
-      text-align: start;
-    }
+      & > header {
+        height: 64px;
+        font-size: 20px;
+        text-align: start;
+      }
 
-    & > section {
-      border-top: 1px solid ${Colors.neutral[5]};
-      border-bottom: 1px solid ${Colors.neutral[5]};
-    }
+      & > section {
+        border-top: 1px solid ${Colors.neutral[5]};
```

Comment:
Why didn't you take it from theme?

Reply:
@Zorii4 will fix it in his own PR

Reply:
Bcs @Zorii4 should be do this fix, it`s not a my bussiness